dsa/src/lib.rs

```diff
@@ -182,8 +190,8 @@ impl TryFrom<&[u8]> for Signature {
     }
 }
 
-/// Returns a `BigUint` with the value 2
+/// Returns a `BoxedUint` with the value 2
 #[inline]
-fn two() -> BigUint {
-    BigUint::from(2_u8)
+fn two() -> BoxedUint {
+    BoxedUint::from(2_u8)
 }
```

Comment:
A `Uint` constant could possibly work here, depending on usage

Reply:
This is used to compare against components and those are BoxedUint, I would need an `impl PartialOrd<Uint> for BoxedUint` for that.

Reply:
Seems doable